File: Uncensorapi/uncensorapi.php
<?php
$url = 'https://goodbyesyntax-uncensorapi.glitch.me/uncensor';
$data = json_encode(["text" => "Th1s 1s @ test"]);

$options = [
    'http' => [
        'header'  => "Content-Type: application/json\r\n",
        'method'  => 'POST',
        'content' => $data,
    ],
];

$context = stream_context_create($options);
$result = file_get_contents($url, false, $context);

if ($result !== FALSE) {
    $response = json_decode($result, true);
    echo $response['uncensored'];
}
?>

File: Uncensorapi/uncensorapi.rs
use reqwest::Client;
use serde_json::json;

#[tokio::main]
async fn main() -> Result<(), reqwest::Error> {
    let client = Client::new();
    let res = client
        .post("https://goodbyesyntax-uncensorapi.glitch.me/uncensor")
        .json(&json!({ "text": "Th1s 1s @ test" }))
        .send()
        .await?
        .json::<serde_json::Value>()
        .await?;

    println!("{}", res["uncensored"]);
    Ok(())
}
